VRoidHubLoader/Modules/VrmLoaderModule.cs

```diff
@@ -62,17 +63,37 @@ private void Update()
         {
             Core.Msg("OnUpdate: VrmLoaderModule F4 pressed");
 
-            // MenuManager is a singleton? sweet.
-            if (!MenuManager.Instance.IsOpen)
+            await ImportVRM();
+        }
+    }
+
+    [HideFromIl2Cpp]
+    public async Task ImportVRM()
+    {
+        string? path = await Core.FileHelper.OpenFileDialog();
+
+        if (!string.IsNullOrEmpty(path))
+        {
+            string fileName = Path.GetFileName(path);
+            string destination = VrmFolderPath + '\\' + fileName;
+            if (!File.Exists(destination))
```

Comment:
Isn't it going to be confusing to the user if they import a model but nothing happens? Would it be safe to replace the model, or we could also make a new one with a suffix added to differentiate them like (1), (2) and so on? But that is quite a bit more complex and maybe we just want to replace.

Reply:
I've been talking with @YusufOzmen01 about this, the loading functionality will most likely be restored here. Although I think the proper way to do this would be to just ignore the load if the model is already selected.

Reply:
My worry is with updated models, those would have the same name, but have different content. The user would probably get really confused if they select the new model but then it doesn't have any of the changes they made, and they might think it is a problem on their end.

Also, even if the original loading functionality gets removed, I do really like the idea of also adding it to the VRM folder, that seems really useful.

Reply:
If the original loading functionality gets restored, not removed, clearly posted a bit too quickly.

Reply:
Restored functionality in [0a993f6](https://github.com/YusufOzmen01/desktopmate-custom-avatar-loader/pull/82/commits/0a993f61a4360f3c4ba28ecc3c82f711fe4b98cb)